Documentation/ApiOverview/Events/Events/FrontendLogin/_ModifyRedirectUrlValidationResultEvent/_ValidateRedirectUrl.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Vendor\MyExtension\EventListener;
+
+use TYPO3\CMS\Core\Attribute\AsEventListener;
+use TYPO3\CMS\FrontendLogin\Event\ModifyRedirectUrlValidationResultEvent;
+
+final class ValidateRedirectUrl
```

Comment:
```suggestion
final readonly class ValidateRedirectUrl
```

Reply:
None of the other Event Listeners I looked at is marked as readonly. Should we mass-change them?

Reply:
Hmm, thought I've already done that. Will do.